lib/auth/apiserver.go

```diff
@@ -69,6 +69,8 @@ func NewAPIServer(config *APIConfig) http.Handler {
 	srv.DELETE("/:version/authorities/:type/:domain", srv.withAuth(srv.deleteCertAuthority))
 	srv.GET("/:version/authorities/:type/:domain", srv.withAuth(srv.getCertAuthority))
 	srv.GET("/:version/authorities/:type", srv.withAuth(srv.getCertAuthorities))
+	srv.POST("/:version/authorities/:type/:domain/activate", srv.withAuth(srv.activateCertAuthority))
```

Comment:
I don't think you should expose these methods as high level API methods. These should be low level side effects of turning cluster on and off